addons/account/models/account_tax.py

```diff
@@ -1293,8 +1368,8 @@ def _prepare_base_line_tax_details(
         else:
             rounding_method = 'round_per_line'
         currency = base_line['currency'] or company.currency_id
-        tax_values_list = taxes._convert_to_dict_for_taxes_computation()
-        taxes_computation = self.env['account.tax']._prepare_taxes_computation(
+        tax_values_list = self._convert_to_dict_for_taxes_computation()
```

Comment:
Are we sure about this change? In that case, shouldn't we use `taxes` as we're iterating on it?